src/ctia/auth/jwt.clj

```diff
@@ -55,15 +55,6 @@
              (throw (ex-info err-msg {:jwt-pubkey-map txt})))))))
 
 
-(defn jwt-error-handler
-  "Return an `unauthorized` HTTP response
-  and log the error along debug infos"
-  [error-msg infos]
-  (let [err {:error :invalid_jwt
-             :error_description error-msg}]
-    (log/info error-msg (pr-str (into infos err)))
```

Comment:
Is the logging still performed after this removal?

Reply:
Yes. The default handler is exactly this, except it doesn't hardcode json response string (seems like a workaround of this very issue).